test/class/opal_fifo.c

```diff
@@ -18,6 +18,7 @@
 #include "opal/runtime/opal.h"
 #include "opal/constants.h"
 
+#include <pthread.h>
```

Comment:
Why are the tests not translated to the MCA threading ?

Reply:
`opal_fifo` and `opal_lifo` were changed use this MCA/thread framework.